apisix/plugins/opentelemetry.lua

```diff
@@ -176,7 +176,7 @@ local schema = {
 
 local _M = {
     version = 0.1,
-    priority = 12009,
+    priority = 11009,
```

Comment:
I think it's not a good idea to set it adjacent to the request-id plugin. It's not extendable if we need to insert another plugin between them in the future.

Reply:
> I think it's not a good idea to set it adjacent to the request-id plugin. It's not extendable if we need to insert another plugin between them in the future.

OK, aready change to 11005